### ISSUE TITLE
Configure Let's Encrypt to auto-renew SSL certificate

### ISSUE THREAD

**Issue by reddog** [healthlocker/healthlocker#1063]
Users need to be able to continue to access a secure site with valid SSL certificate at all times.

Is certbot (Let's Encrypt agent) configured to auto-renew SSL certificates?  The **current certificate is due to on 11/10/17.**

![image](https://user-images.githubusercontent.com/151362/30156181-e0ffe9b6-93b6-11e7-83f3-dc56f47e3b2b.png)

Let's Encrypt certificates expire after 90 days. Once a website certificate expires, accessing that site will present a browser message strongly worded such that the user should not continue using the site.

certbot comes with an easy to configure cron-able utility to auto-renew certificates, see https://certbot.eff.org/#ubuntuxenial-nginx

> you can arrange for automatic renewal by adding a cron or systemd job which runs the following:
```
certbot renew
```
certbot auto-renewal should be configured for test (staging) site.  Added to deployment plan for live site.

**Comment by RobStallion** [comment]
![image](https://user-images.githubusercontent.com/15571853/36536479-f02b96a0-17c4-11e8-8f33-8851d43396d4.png)

Certificate is back up now. 

Steps taken to resolve issue
 - systemctl stop nginx.service
 - certbot renew
 - systemctl start nginx.service

However, when I ran the command `certbot renew` I got the following message 
```
The following certs are not due for renewal yet:
  /etc/letsencrypt/live/staging.healthlocker.uk/fullchain.pem (skipped)
No renewals were attempted.
```

This leads me to believe that the `renew` command had already been run but that `nginx` was not picking up the new certificate that had been created until it was restarted. 

Will look into the `cron job` that is being run to see if there is an issue. Cron job can be found in the file `/etc/cron.d/certbot`
